nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/event/test/TransitionTest.java

```diff
@@ -105,4 +136,23 @@ public void initListener() {
         });
     }
 
+    protected void unregisterAsEventListenerFor(final String eventName) {
+        final EventListener el = this;
+        eventService.removeEventListener(new EventListenerDescriptor() {
```

Comment:
This is not going to remove the `EventListenerDescriptor` previously added. The `registerAsEventListenerFor` method must keep the instance of `EventListenerDescriptor` it has added, and this `unregisterAsEventListenerFor` must remove the same instance.